rollup.config.ts

```diff
@@ -15,5 +15,5 @@ export default {
     }
   ],
   plugins: [typescript()],
-  external: ["axios"]
+  external: ["axios", "form-data", "stream"]
```

Comment:
I think it's because I'm only importing `IncomingMessage` from `http` which I'm just using for type information. `http` isn't actually required in the rolled up code, which just has these three imports:

```
import axios from 'axios';
import { PassThrough, Readable } from 'stream';
import FormData from 'form-data';
```